### DISCUSSION TITLE
[Newbie] Looking For Advice on Local BitWarden Installation on Windows

### DISCUSSION THREAD

**Discussion by Infinitay** [dani-garcia/vaultwarden#4000] [Q&A]
I've always wanted to give BitWarden a try but the setup process is intimidating to me because I have never configured proxies (nginx, caddy, etc.), ssl certificates, or a proper dns resolver (something like when visiting <my_network>.local it will redirect to your localhost). When I take a look at the wiki or YouTube videos, I see people messing about their proxy files, getting to parts about setting up SSL and talking about using their existing reverse proxy or something. I haven't done any of that and I was wondering if I would be able to setup BitWarden locally on my second PC and simply host it and connect to it via my second pc's ip?

Also for the sake of asking, if I did want to host it such that I could access my BitWarden anywhere outside my local network, would I need to do anything extra for proper security when exposing the port?

**Comment by Masgalor** [comment]
Just to be clear, you want to install vaultwarden right?
If you want to install bitwarden this is the wrong place to look for help.

The short answer is: yes, you can use your spare windows pc to run vaultwarden.
But that does not lower the requirements, the opposite is true, running docker on windows is way more complicated than on other systems.

If the wiki section about a deployment with docker-compose does not look doable for you, I am afraid to say it does not get much simpler then that.
[Link](https://github.com/dani-garcia/vaultwarden/wiki/Using-Docker-Compose)

Some hosting companies like hetzner or linode might offer vaultwarden oder bitwarden as ready to go applications.

**Reply by Infinitay** [reply]
Yes I want to install vaultwarden. I don't have a problem with deploying via Docker, but rather how it is meant to be deployed. For example, I don't have a domain because I don't intend to make any of my self-hosted apps available on the internet. I just want everything to be local. That's why I am confused with all the tutorials I see that involve tunneling or reverse proxies.
